engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/ReplTest.scala

```diff
@@ -102,7 +102,7 @@ class ReplTest
           result.toString shouldEqual "Error in method `to_text` of [Bar 1]: Expected Text but got 42"
         }
         inside(executor.evaluate("C.Baz 1")) { case Right(result) =>
-          result.toString shouldEqual "Error in method `to_text` of [Baz 1]: Expected Text but got C.to_text[Test:18:1-29]"
+          result.toString shouldEqual "Error in method `to_text` of [Baz 1]: Expected Text but got C.to_text[Test:18:1-28]"
```

Comment:
The relevant line indeed has _only_ 28 characters, so I think that this is an evidence that the fix of the `+1` offset was correct.